gulpfile.js

```diff
@@ -4,7 +4,7 @@ var gulp = require("gulp");
 var gulp_tslint = require("gulp-tslint");
 
 gulp.task("tslint", () => {
-    return gulp.src(['**/*.ts', '!**/*.d.ts', '!node_modules/**'])
+    return gulp.src(['src/**/*.ts', '!**/*.d.ts', '!node_modules/**'])
```

Comment:
Why is it related to the additional tests?

Reply:
it is not directly related but I keep it here.

Reply:
what is the goal of the modification then?

Reply:
ah finally removed, my bad